pkg/virt-handler/container-disk/mount_test.go

```diff
@@ -54,7 +53,9 @@ var _ = Describe("ContainerDisk", func() {
 	BeforeEach(func() {
 		tmpDir, err = os.MkdirTemp("", "containerdisktest")
 		Expect(err).ToNot(HaveOccurred())
-		vmi = api.NewMinimalVMI("fake-vmi")
+		vmi = libvmi.New(
+			libvmi.WithName("fake-vmi"),
+		)
 		vmi.UID = "1234"
```

Comment:
Same. If there is not already an option for `UID` it might make sense to add one.